docs/docs/userguide/PREREQUISITES.md

```diff
@@ -4,38 +4,42 @@
 In order to deploy Atlassian’s Data Center products, the following is required:
 
 1. An understanding of [Kubernetes](https://kubernetes.io/docs/concepts/overview/what-is-kubernetes/){.external} and [Helm](https://helm.sh/){.external} concepts
-2. A Kubernetes cluster, running Kubernetes `v1.19` or later
 3. `kubectl` `v1.19` or later, must be compatible with your cluster
 4. `helm` `v3.3` or later
```

Comment:
Can we hyperlink these two to their install instructions

Reply:
Like this?

<img width="664" alt="Screen Shot 2021-08-31 at 9 25 08 am" src="https://user-images.githubusercontent.com/416238/131418454-3976dec5-fb87-4761-b468-6a7adbe1bd39.png">